libs/blocks/footer/footer.css

```diff
@@ -304,9 +304,6 @@ footer .footer-region-button.inline-dialog-active svg.icon-chevron-down {
 }
 
 /* PRIVACY INFO */
-.footer-privacy {
-  font-size: var(--type-detail-s-size);
-}
```

Comment:
This file is used just for older pages, are you sure it achieves your desired result? It might be that we need to make some metadata changes to the page if that's the case. Same comment for L15

Reply:
Good catch! It seems I was experiencing a rendering issue locally, where this original file for the footer was being displayed instead of the global footer. That would explain why I wasn't seeing the line height issue I thought I fixed 🙃. Restarting my server looked like it fixed the issue.